src/Epicea/TestCase.extension.st

```diff
@@ -1,14 +1,14 @@
 Extension { #name : 'TestCase' }
 
 { #category : '*Epicea' }
-TestCase >> shouldLogWithEpicea [
+TestCase class >> shouldLogWithEpicea [
+	"When running a test, we disable the loggings of Epicea to run the tests silently. In case a TestCase explicitly wants to Epicea logs, then I can be overriden to return true and enable the logs."
 
-	^ self class shouldLogWithEpicea
+	^ false
```

Comment:
Why do you disable this feature?
May be you did it in your image for you?

Reply:
Mmm I don't remember to changed that code.
I will review the changes now

Reply:
It could be an automated refactoring?
I've seen a lot of them automatically applied so I have to uncheck changes in Iceberg

Reply:
It’s just the instance and class side methods of the same name that got inverted. That happens sometimes and seems to be a bug in tonel. I’ve it on my todo list to check it but never got the time. So there is no problem here